### ISSUE TITLE
Improve accessibility for MediaOverlay component hover state

### ISSUE THREAD

**Issue by simast** [wix/wix-style-react#4421]
# 🐛 Bug Report

### 🏗 Relevant Components

- `<MediaOverlay/>`
- `<CardGalleryItem/>`
- `<ImageViewer/>`

### 😯 Current Behavior

Users cannot access actions available in hover state using keyboard only.

### 🤔 Expected Behavior

1. First TAB press would make the component focused (and would reveal/activate hover state immediately)
2. Next TAB press would focus any revealed focusable elements in a hover state.

If there is no hover state content - we should not make the component focusable.

### 👀 Severity

- Low

**Comment by shlomitc** [comment]
won't be fix
